code/modules/mob/living/carbon/xenomorph/Castes/Hunter.dm

```diff
@@ -152,8 +152,8 @@
 	plasma_stored = 50
 	tier = 2
 	upgrade = 0
-	pixel_x = -12
-	old_x = -12
+	pixel_x = 0
```

Comment:
It should be safe to just remove both as I couldn't find them redefined at the Xenomorph level.

Reply:
changed